account_move_accounting_date_tree/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Invoice - Accounting Date on tree view",
+    "summary": """
+        Shows Accounting Date on Invoices tree views
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.2.0.0",
```

Comment:
```suggestion
    "version": "17.0.1.0.0",
```
Migration should have a fresh start